app/src/main/java/org/fossasia/openevent/general/di/Scopes.kt

```diff
@@ -0,0 +1,21 @@
+package org.fossasia.openevent.general.di
+
+/**
+ * Enum class to collect all possible Fragment scopes for Koin DI
+ * in one place. This list is expected to grow as Scopes are used in more
+ * fragments.
+ */
+enum class Scopes {
+    EVENTS_FRAGMENT {
+        override fun toString() = "events_fragment"
```

Comment:
No need to do this

Reply:
I'm not sure what you mean by this. The scopes enum is not needed? Or is there some other problem?

Reply:
I implemented an enum class because the scopes feature of Koin requires an associated String tag with each scope. I thought that if scopes would be used more widely in the app in the future, then having all possible values in the form of constants in one location would be very convenient, as it would help make sure that no two scopes share the same string tag value. If you feel that an enum is not necessary, I can remove it and replace the string tags in each scope with a direct string value.

Reply:
I think you don't know that you don't need to override toString in enums? All my reviews have been to remove redundant property and toString overrides. Enums already have string value associated with them, it's their name itself. Read about enums please

Reply:
https://dzone.com/articles/enum-using-the-name-and-tostring-methods-correctly

Reply:
I did not know that Enums have an associated string value by default. I feel kinda stupid now 😓

Thanks though. Updating the PR in a few mins.

Reply:
Removed the overridden `toString()` method. Things looks a lot a better now, thanks.